### PR TITLE
added abracadabra, fei, frax, and usdp (under paxos folder)

### PR DESCRIPTION
**Pull request by davidlwg**

### CODE REVIEW

File: dags/resources/stages/parse/table_definitions/abracadabra/MagicInternetMoneyV1_event_Approval.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            ],
+            "name": "Approval",
+            "type": "event"
+        },
+        "contract_address": "0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "ethereum_mim",
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/abracadabra/MagicInternetMoneyV1_event_Approval.json:33]: The dataset name should match the folder name

File: dags/resources/stages/parse/table_definitions/abracadabra/MagicInternetMoneyV1_event_OwnershipTransferred.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            ],
+            "name": "OwnershipTransferred",
+            "type": "event"
+        },
+        "contract_address": "0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "ethereum_mim",
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/abracadabra/MagicInternetMoneyV1_event_OwnershipTransferred.json:27]: The dataset name should match the folder name

File: dags/resources/stages/parse/table_definitions/fei_protocol/Fei_event_Burning.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            ],
+            "name": "Burning",
+            "type": "event"
+        },
+        "contract_address": "0x956f47f50a910163d8bf957cf5846d573e7f87ca",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "ethereum_fei",
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/fei_protocol/Fei_event_Burning.json:33]: The dataset name should be same as folder name

Comment (medvedev1088) [reply dags/resources/stages/parse/table_definitions/fei_protocol/Fei_event_Burning.json:33]: Let's make it `fei`

File: dags/resources/stages/parse/table_definitions/frax_protocol/FRAXStablecoin_event_Approval.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            ],
+            "name": "Approval",
+            "type": "event"
+        },
+        "contract_address": "0x853d955acef822db058eb8505911ed77f175b99e",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "ethereum_frax",
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/frax_protocol/FRAXStablecoin_event_Approval.json:33]: Should be `frax`. Folder name should be `frax` too

File: dags/resources/stages/parse/table_definitions/paxos/USDPImplementationV3_event_Approval.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            ],
+            "name": "Approval",
+            "type": "event"
+        },
+        "contract_address": "0x8e870d67f660d95d5be530380d0ec0bd388289e1",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "ethereum_usdp",
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/paxos/USDPImplementationV3_event_Approval.json:30]: Should be same as folder name

**Comment by medvedev1088** [comment]
Please remove the .DS_Store file

**Comment by davidlwg** [comment]
> Please remove the .DS_Store file

I removed this file in the original commit, unless I'm missing another file somewhere.

Screen Shot 2022-05-21 at 8 23 29 PM

**Comment by medvedev1088** [comment]
> > Please remove the .DS_Store file
> 
> I removed this file in the original commit, unless I'm missing another file somewhere.
> 
> Screen Shot 2022-05-21 at 8 23 29 PM

Sorry my bad

**Pull request merged**